tests/components/camera/test_demo.py

```diff
@@ -67,19 +67,6 @@ def demo_camera(hass):
     assert demo_camera.state == STATE_STREAMING
 
 
-async def test_turn_off_unsupport_camera(hass, demo_camera):
```

Comment:
This test has been removed because this was the only service that checks supported features before calling a method. Removed it for consistency. Architecture issue can be opened to explore if this is a wanted feature.